de.lucaswerkmeister.corebird.json

```diff
@@ -0,0 +1,67 @@
+{
+  "app-id": "de.lucaswerkmeister.corebird",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "corebird",
+  "rename-icon" : "corebird",
+  "copy-icon" : true,
+  "finish-args" : [
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--filesystem=home:ro",
+    "--filesystem=~/.corebird/:rw",
+    "--filesystem=~/.config/corebird/:create",
```

Comment:
You don't need direct access to this directory either.

Reply:
Same question here.

Reply:
Flatpak overrides `XDG_CONFIG_HOME` so it simply doesn't write here.

Reply:
Ah, I had been wondering why there’s no such thing as `--filesystem=xdg-config` or similar (compared to `--filesystem=xdg-desktop` etc.) – that explains it :)